### ISSUE TITLE
"Failed to retrieve Google event" x 3 every sync

### ISSUE THREAD

**Issue by jbrams** [phw198/OutlookGoogleCalendarSync#510]
v2.7.2.0  
Installed

Issue:
Each time I sync I get this error three times: "Failed to retrieve Google event"
The sync still finishes successfully, but I'd like to resolve this error. 

I believe this is the relevant [section of the log](https://gist.github.com/phw198/b9a506dbedb6c72be6f55ff31e0f56cd).

**Comment by dpfulg** [comment]
I get a similar issue here. Two events can not be retrieved from Google calendar. 

Unfortunately the log file does only show an ID that I cannot match to any of my events...
```
2018-05-08 06:54:48,373 DEBUG  8 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [0] -  Retrieving specific Event with ID q01co55ce5rkbl4gstq5rb0ka0
2018-05-08 06:54:48,416 WARN  8 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [0] -  Google.Apis.Requests.RequestError
Not Found [404]
Errors [
	Message[Not Found] Location[ - ] Reason[notFound] Domain[global]
]
```

**Comment by jeffwash** [comment]
I'm having the same issue

**Comment by phw198** [comment]
Thanks for reporting. 

:package: Could you upgrade to v2.7.3 and then apply [v2.7.3.1.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/1998967/v2.7.3.1.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)) - this should add some extra logging.

Having started v2.7.3.1, please increase the logging level to `FINE` and then replicate the error. You should then see in your `OGcalsync.log` file a new line like the first one below:
```
Found a master Outlook recurring item outside sync date range: <appointment details>
Retrieving specific Event with ID 4er5it0lut2br57n1mgofd48di
```
So for that appointment, is there a reason you know why it exists in Outlook, but not Google? Did you delete the entire recurring series in Google, or maybe just the first occurrence? Might it have been an invite from someone else, and they have then cancelled the meeting?

Please upload your `OGcalsync.log` file again and any further information would be helpful.

Thanks!

**Comment by jeffwash** [comment]
Thanks Paul,

I’ve applied the update.  I’ll let you know how it goes.

BTW, I donated a few versions ago, how do I get rid of the splash screen?

Thanks,
-Jeff

**Comment by phw198** [comment]
Any updates? I thought the error happened on every sync, so would be easy to produce the logfile?

**Comment by jeffwash** [comment]
Hi,

Everything seems to be syncing fine now with this latest version.

Thanks,
-Jeff

**Comment by phw198** [comment]
Huh, maybe this was already fixed in v2.7.3 then...

**Comment by phw198** [comment]
:package: This should now be resolved with [v2.7.5.6.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/2299364/v2.7.5.6.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)) - ensure it is applied over v2.7.5.x.

Let me know how it goes.
